### ISSUE TITLE
Certificate has less than 30 days validity, but isn't being refreshed

### ISSUE THREAD

**Issue by toothbrush** [Celti/lets-nfsn.sh#6]
Hello!  Today i received another email from NFS cron saying:

```
Your scheduled task "letsencrypt" on site xxx:

/home/private/lets-nfsn.sh/nfsn-cron.sh

completed at 2016-08-16 06:00:42 UTC after 2 seconds and produced the
following output:

+ Updating lets-nfsn.sh...
Already up-to-date.
 + Updating letsencrypt.sh...
 + Checking certificate expiration date...
 + More than 30 days until any certificate expires. Exiting.
```

However, when i manually visit my site, click the padlock and View Certificate in my browser, i see that its expiry is on the 12th of September, which seems like less than 30 days from now.  What could be the issue?  On occasion i've seen a successful `git pull` in the cron report, so i believe i'm running the latest version of this script.

Please don't hesitate to ask if i can provide more info, i'm not really sure how to debug this.  Apologies in advance if it's PEBKAC!  Thanks :)

**Comment by ghost** [comment]
Confirming here. My certificate already expired but cron job still says "more than 30 days remaining". The detection function doesn't seem to work properly as the find command does indeed output "Certificate will expire".

But even if I run the commands manually, certificates are not getting updated. This may be let's encrypt's fault though.

**Comment by jbranso** [comment]
I can also confirm that I am getting the same error.

**Comment by chludek** [comment]
I had exactly the same problem. 
After a little research, I've found the following post by Celti on the NFSN forum:

> Sadly, it looks like I'm going to have to stop maintaining this [lets-nfsn.sh]. 
> [Source](https://members.nearlyfreespeech.net/forums/viewtopic.php?p=44297#44297)

So, I don't think that he/she will fix the bug.

I just followed the [guide by aleksandyr](https://members.nearlyfreespeech.net/forums/viewtopic.php?t=8457) to generate new let's encrypt certifications and added them via the "Upload TLS Files" function to my site. Not as simple as lets-nfsn.sh, but it worked out great 😄
